airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -323,24 +324,26 @@ private void reportSuccess(final ConnectionUpdaterInput connectionUpdaterInput,
 
   private void reportFailure(final ConnectionUpdaterInput connectionUpdaterInput,
                              final StandardSyncOutput standardSyncOutput,
-                             final FailureCause failureCause) {
+                             final FailureCause failureCause,
+                             final Set<FailureReason> failureReasonsOverride) {
```

Comment:
Since most of the calls here simply pass in `new HashSet<>()`, should we have a method that doesn't require the `failureReasonsOverride` parameter?

Reply:
Done,